actionpack/lib/abstract_controller/rendering.rb

```diff
@@ -16,8 +16,22 @@ module Rendering
     extend ActiveSupport::Concern
     include ActionView::ViewPaths
 
-    # Normalizes arguments, options and then delegates render_to_body and
+    # Normalizes arguments, options and then delegates <tt>render_to_body</tt> and
     # sticks the result in <tt>self.response_body</tt>.
+    #
+    # The primary options are:
```

Comment:
Will do. I was thinking of removing everything from **the primary options are:** and replacing it with: 

_[Visit the Guides] for a summary of options that can be passed to <tt>render</tt>_

How's that sound to you?